src/components/NavBar.js

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react'
+
+function NavBar() {
+    const [isOpen, setIsOpen] = useState(false)
+    const [scroll, setScroll] = useState(false)
+
+    function handleClick() {
+        setIsOpen(!isOpen)
+    }
+
+    function handleScroll() {
+        window.scrollY >= 546 ? setScroll(true) : setScroll(false)
```

Comment:
Could you tell a little about what were you trying to achieve with `scroll` and `setScroll` variables? It's a little bit hard to understand what they are for.

My guess is that it is for screens with height less than 546. But as far as I know `window.scrollY` would show not the screen hight but scrollable height of the page - and I think with this we will never encounter case where `window.scrollY < 546`

Also it's a good practice to name all your constants so it's easy to understand what is happening if we were to modify this code in a few months :)

Reply:
scrollY is measuring the scroll value .. on page load scrollY = 0 and on scrolling value increases and at some point which is 546 the navbar style changes from dark to white